asset_manager/file_manager/scripts/backup.py

```diff
@@ -0,0 +1,281 @@
+from django.conf import settings
+from datetime import datetime
+
+import os
+import pexpect
+import smtplib
+import sys
+
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+import boto3
+from botocore.exceptions import ClientError
+
+import logging
+logging.basicConfig(
+    filename=settings.LOGFILE,
+    level=logging.INFO,
+    format=' %(asctime)s - %(levelname)s - %(message)s'
+    )
+# logging.disable(logging.CRITICAL)
+
+s3 = boto3.client(
+    's3',
+    aws_access_key_id=settings.AWS_ACCESS_KEY_ID,
+    aws_secret_access_key=settings.AWS_SECRET_ACCESS_KEY)
+
+VENV_DIR = '/Users/georgemillard/programming/projects/virtualenvs/asset_manager_venv/bin/activate'
```

Comment:
@georgemillard Is this removable? Can't see it being used elsewhere and is specific to you ;)